distributor-api-v1/operations.md

```diff
@@ -123,48 +122,150 @@ Preferred initial call used to obtain all static data about distributor configur
     "CurrencyCodes": [],
+    "DisplayVoucherCode": false,
     "EndDateOffset": null,
     "GtmContainerId": "",
     "IntroVideoUrl": "",
     "LanguageCode": null,
-    "NowUtc": "",
+    "NowUtc": "2020-04-09T07:18:48Z",
     "PrimaryColor": "",
     "StartDateOffset": null,
     "Theme": null,
-    "VoucherCode": null,
-    "DisplayVoucherCode": null,
+    "VoucherCode": ""
 }
 ```
 
 |  | Property | Type | Description |
 | :--- | :--- | :--- | :--- |
-| `Cities` | array of [City](operations.md#city) | required | Cities supported by hotel. |
-| `CityId` | string | required | ID of default city. |
-| `Configurations` | array of [Configuration](operations.md#configuration) | required | Configurations matching the configuration IDs in request. |
-| `CurrencyCode` | string | optional | Code of default currency accepted by hotel. |
-| `Currencies` | array of [Currency](operations.md#currency) | required | Currencies accepted by hotel. |
-| `EndDateOffset` | string | optional | TBC |
-| `GtmContainerId` | string | optional | Google tag manager indentifier. |
-| `IntroVideoUrl` | string | optional | TBC |
-| `LanguageCode` | string | optional | Code of default language. |
-| `NowUtc` | string | required | Server UTC date and time. |
-| `PrimaryColor` | string | optional | TBC |
-| `StartDateOffset` | string | optional | TBC |
-| `Theme` | string | optional | TBC |
-| `VoucherCode` | string | optional | TBC |
+| `Cities` | array of [City](operations.md#city) | required | Cities supported by the enterprise. |
+| `CityId` | string | required | Unique identifier of the default city. |
+| `Configurations` | array of [Configuration](operations.md#configuration) | required | Array of [Configuration](operations.md#configuration)s. |
+| `CurrencyCode` | string | optional | ISO 4217 code of the currency which Distributor should use when displaying prices. |
+| `DisplayVoucherCode` | boolean | required | Determines whether enterprise's voucher codes should be listed in Distributor (voucher codes are listed by default). |
+| `StartDateOffset` | string | optional | Number of days after the day that the customer is booking that will be selected as the default start date in the date picker (for example, if `3` is set and a customer uses the booking engine on the 1st day of the month, the default start date will be the 4th). If left blank, the default will be 0. |
+| `EndDateOffset` | string | optional | Number of days after the day that the customer is booking that will be selected as the default end date in the date picker  (for example, if `3` is set and a customer uses the booking engine on the 1st day of the month, the default end date will be the 3rd). If left blank, the default will be `4`. |
+| `GtmContainerId` | string | optional | Google Tag Manager identifier. |
+| `IntroVideoUrl` | string | optional | Distributor's intro video URL. |
+| `LanguageCode` | string | optional | Language code in BCP 47 which Distributor should use. |
```

Comment:
@vitasamek I've tried to find what the standard is and found this through googling. Is it correct?

Reply:
Let's remove the format,  because there are still other places where is not defined. e.g 
|languageCode | Language code, i.e. `en-US`. |

I am sorry for the comment to put it here.

Reply:
will do